RELEASENOTES.md

```diff
@@ -4,6 +4,15 @@ This file contains highlights and announcements covering all components.
 For more details see `CHANGELOG.md` files maintained in the root source
 directory of each individual package.
 
+## Unreleased
```

Comment:
I think this is to be updated during 1.10 update, not required in the interim beta/rc stages?

Reply:
There was a request in the previous PR to highlight the removal of the old target and the addition of the new one. Tracking it this way ensures we won't miss updating this during the actual release. We could update the release scripts to modify it automatically.

Reply:
Agree with @cijothomas. This file should be a curated summary of high-level/interesting things being released in 1.10.0 (stable) which we'll add as part of that final drop. This content being added here seems too low-level/CHANGELOG-ish. If you want, you could add notes here we'll use later to make that content. Something like...

```md
<!--
## 1.10.0

TODO:
* Mention drop of net6.0 & net7.0 targets, addition of net9.0 target.
-->
```

Or you could just do nothing to this file and we can work on the release notes later 🤣

Reply:
Will remove the changes to this file to keep it simple.